src/frontend/mod.rs

```diff
@@ -647,6 +658,45 @@ impl<'a> Frontend<'a> {
         self.buffer.push_back(WithRange(Event::End(tag), range));
     }
 
+    fn convert_block_quote(&mut self, range: SourceRange, mut cskvp: Cskvp<'a>) {
+        let prefix = cskvp.take_single();
+
+        let kind = if let Some(prefix) = prefix {
+            match prefix.as_ref().element().as_ref() {
+                "corollary" => ProofKind::Corollary,
+                "definition" => ProofKind::Definition,
+                "lemma" => ProofKind::Lemma,
+                "proof" => ProofKind::Proof,
+                "theorem" => ProofKind::Theorem,
+                other => {
+                    self.diagnostics
+                        .error(format!("unknown quote environment {:?}", other))
+                        .with_error_section(prefix.range(), "type defined here")
+                        .emit();
+                    return;
+                }
+            }
+        } else {
+            self.diagnostics
+                .error("quote environment has no type")
+                .with_error_section(range, "configuration defined here")
+                .help("try indicating the type, the configuration could look like")
+                .help("  {proof}")
+                .emit();
+            return;
+        };
```

Comment:
What if I want to have a quote within a figure with a caption, or a quote with a label? We don't support that right now, but with a change like this, I think we should support it in the long run. If you don't want to change it right now, could you please open an issue?